analyzer/codechecker_analyzer/cmd/fixit.py

```diff
@@ -225,6 +249,10 @@ def list_fixits(inputs, checker_names, file_paths, interactive, reports):
         print("Not existing files:\n{}".format(
             '\n'.join(sorted(not_existing_files))),
             file=sys.stderr)
+    if modified_files:
+        print("Skipped files due to modification since last analysis:\n{}"
```

Comment:
Why do you use `print` instead of `LOG.error` here?

Maybe we can print some more information what the user can do to eliminate these messages: `Please analyze your project again to update fixits!`